python/spear/engine_service.py

```diff
@@ -0,0 +1,73 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+import spear
+import sys
+
+# TODO (RP): Separate EngineService and GameWorldService function calls from this class.
+class EngineService():
+    def __init__(self, rpc_client):
+        self._rpc_client = rpc_client
+
+    def call(self, service_name, func_name, *args):
+        return self._rpc_client.call(service_name + "." + func_name, *args)
+
+    def begin_tick(self):
```

Comment:
Don't compose functions like this in `EngineService`. Higher-level systems (e.g., `Env`) should be responsible for composing.